include/clang/AST/Type.h

```diff
@@ -1917,6 +1917,9 @@ class Type : public ExtQualsTypeCommonBase {
   /// \brief Whether this type is or contains a checked type
   bool isOrContainsCheckedType() const;
 
+  /// \brief Whether this type(Struct/Union) contains a checked value
```

Comment:
I would word this slightly more precisely: check whether an object of a struct or union type contains a checked value.